build.gradle

```diff
@@ -18,6 +18,7 @@ plugins {
   id 'org.owasp.dependencycheck' version '5.2.4'
```

Comment:
Can you please remove all the changes which are not related to Unit and integration testing elements.

Reply:
Added comments in build.gradle file for only changes related to Integration tasks.